src/test/java/com/wnis/linkyway/controller/CardControllerTest.java

```diff
@@ -76,4 +83,70 @@ void addCardSuccess() throws Exception {
                                            .andReturn();
     }
 
+    @Nested
+    @DisplayName("단일 북마크(카드) 상세 조회")
+    class findCardByCardId {
+
+        Long cardId;
+        CardResponse cardResponse;
+
+        @BeforeEach
+        void setCard() {
+            cardId = 3L;
+            cardResponse = CardResponse.builder()
+                                       .cardId(cardId)
+                                       .link("https://www.naver.com/")
+                                       .title("title1")
+                                       .content("content1")
+                                       .shareable(true)
+                                       .build();
+        }
+
+        @Test
+        @DisplayName("상세 조회 성공: 올바른 URL")
+        void CardExistFindingSuccess() throws Exception {
+            // given
+            doReturn(cardResponse).when(cardService).findCardByCardId(cardId);
+
+            // when
+            ResultActions resultActions = mockMvc.perform(get("/api/cards/"
+                    + cardId).contentType("application/json")
+                             .content(objectMapper.writeValueAsString(
+                                     cardResponse)));
+
+            // then
+            MvcResult mvcResult = resultActions.andExpect(status().isOk())
+                                               .andExpect(
+                                                       ResponseBodyMatchers.responseBody()
+                                                                           .containsPropertiesAsJson(
+                                                                                   Response.class))
+                                               .andReturn();
+            verify(cardService).findCardByCardId(Mockito.anyLong());
+        }
+
+        @Test
+        @DisplayName("상세 조회 실패: 잘못된 URL")
+        void CardNotExistFindingFail() throws Exception {
+            // given
+            doReturn(cardResponse).when(cardService)
+                                  .findCardByCardId(Mockito.anyLong());
+
+            // when
+            ResultActions resultActions = mockMvc.perform(
+                    get("/api/cards/").contentType("application/json")
```

Comment:
```suggestion
                    patch("/api/cards/").contentType("application/json")
```

api 에 없는 http 메소드로 요청해보시면 405 나옵니당